frontend/apps/marketing/src/themes/code.org/index.ts

```diff
@@ -13,11 +13,48 @@ const theme = createTheme({
       styleOverrides: {
         root: {
           ['&.MuiButton-contained.MuiButton-colorPrimary']: {
-            backgroundColor: '#8c52ba',
+            backgroundColor: 'var(--brand-purple-50)',
           },
         },
       },
     },
+    MuiLink: {
+      styleOverrides: {
+        root: {
+          color: 'var(--text-brand-purple-primary)',
+          fontWeight: 500,
+          textDecoration: 'underline',
+          transition: 'color 0.2s ease-in-out',
+          '&:hover': {
+            color: 'var(--text-brand-purple-secondary)',
+            '& svg': {
+              color: 'var(--text-brand-purple-secondary)',
+            },
+          },
+          ['& svg']: {
+            color: 'var(--text-brand-purple-primary)',
+            transition: 'color 0.2s ease-in-out',
+          },
+        },
+      },
+    },
+    MuiTypography: {
+      styleOverrides: {
+        root: {
+          color: 'var(--text-neutral-primary)',
+        },
+      },
+    },
+  },
+  typography: {
+    fontFamily: ['Figtree', 'Noto Sans', 'sans-serif'].join(','),
```

Comment:
Slight nit to avoid the `join(',')` as it's slightly more performant. Also, the fonts need to include the other fonts we support:


```suggestion
    fontFamily: 'Figtree, Noto Sans, Noto Sans Math, Noto Sans Arabic, Noto Sans Armenian, Noto Sans Bengali, Noto Sans SC, Noto Sans TC, Noto Sans Devanagari, Noto Sans Georgian, Noto Sans Hebrew, Noto Sans JP, Noto Sans Kannada, Noto Sans Khmer, Noto Sans KR, Noto Sans Myanmar, Noto Sans Sinhala, Noto Sans Tamil, Noto Sans Telugu, Noto Sans Thai, Noto Sans Thaana, sans-serif;'
```